cmake/std/atdm/ctest-s-local-test-driver.sh

```diff
@@ -43,16 +43,16 @@ driver script name is:
   Trilinos/cmake/ctest/drivers/atdm/<system_name>/drivers/
     ${ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX}<build-name-keysi>.sh
 
-For example, ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-waterman- and
+For example, ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-weaver- and
```

Comment:
We need to change this documentation to use a different example than 'waterman' or 'weaver'.  I think we could pick just about any other supported system.

Reply:
Replaced with "ats2"